runtime/frequency/src/lib.rs

```diff
@@ -1647,6 +1682,46 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+	pub MbmServiceWeight: Weight = Perbill::from_percent(80) * RuntimeBlockWeights::get().max_block;
+}
+
+impl pallet_migrations::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type Migrations = (
+		pallet_messages::migration::MigrateV2ToV3<
+			Runtime,
+			pallet_messages::weights::SubstrateWeight<Runtime>,
+		>,
+		pallet_messages::migration::FinalizeV3Migration<
+			Runtime,
+			pallet_messages::weights::SubstrateWeight<Runtime>,
+		>,
+		pallet_stateful_storage::migration::v2::MigratePaginatedV1ToV2<
+			Runtime,
+			pallet_stateful_storage::weights::SubstrateWeight<Runtime>,
+		>,
+		pallet_stateful_storage::migration::v2::MigrateItemizedV1ToV2<
+			Runtime,
+			pallet_stateful_storage::weights::SubstrateWeight<Runtime>,
+		>,
+		pallet_stateful_storage::migration::v2::FinalizeV2Migration<
+			Runtime,
+			pallet_stateful_storage::weights::SubstrateWeight<Runtime>,
+		>,
+	);
```

Comment:
Are the order of these migrations important? I think if they are maybe we want the storage to be first since we are locking the functionality.